war/src/test/java/io/lumeer/engine/rest/SecurityServiceIntegrationTest.java

```diff
@@ -0,0 +1,1413 @@
+      });
+      groups.forEach((s)-> {
+         if (groupCount.containsKey(s)) groupCount.put(s, groupCount.get(s) + 1);
+         else groupCount.put(s, 1);
+      });
+      userCount.forEach((k, v) -> {
+         assertThat(v == 1).isTrue();
+      });
+      groupCount.forEach((k, v) -> {
+         assertThat(v == 1).isTrue();
+      });
+   }
+
+   @Test
+   public void testGetOrganizationRolesNoQueryParam() {
+      final List<Role> before = securityFacade.getOrganizationRoles(organizationCode);
+
+      final Client client = ClientBuilder.newBuilder().build();
+      final String role = LumeerConst.Security.ROLE_MANAGE;
+
+      Response response = client
+            .target(TARGET_URI)
+            .path(createUrl(
+                  LumeerConst.Security.ORGANIZATION_RESOURCE,
+                  organizationCode,
+                  LumeerConst.Security.ROLES_KEY,
+                  role))
+            .request().build(PUT_METHOD_NAME)
+            .invoke();
+      assertThat(response.getStatus()).isEqualTo(Response.Status.NO_CONTENT.getStatusCode());
+
+      final List<Role> after = securityFacade.getOrganizationRoles(organizationCode);
+
+      assertThat(before.size()).isEqualTo(after.size());
+      for (Role r : before) {
+         assertThat(after.contains(r));
+      }
+   }
+
+   @Test
+   public void testGetOrganizationRolesBasic() {
+      final String role = LumeerConst.Security.ROLE_MANAGE;
+
+      securityFacade.addOrganizationUserRole(organizationCode, USER_Y, role);
+      securityFacade.addOrganizationGroupRole(organizationCode, GROUP_B, role);
+      List<Role> fromFacade = securityFacade.getOrganizationRoles(organizationCode);
+
+      final Client client = ClientBuilder.newBuilder().build();
+      Response response = client
+            .target(TARGET_URI)
+            .path(createUrl(
+                  LumeerConst.Security.ORGANIZATION_RESOURCE,
+                  organizationCode,
+                  LumeerConst.Security.ROLES_KEY))
+            .request().buildGet()
+            .invoke();
+
+      List<Role> fromService = response.readEntity(List.class);
+      assertThat(fromFacade.size()).isEqualTo(fromService.size());
+      for (Role r : fromFacade) {
+         assertThat(fromService.contains(r));
+      }
+
+      response.close();
+   }
+
+   @Test
+   public void testGetOrganizationRolesNothingAdded() {
+      List<Role> fromFacade = securityFacade.getOrganizationRoles(organizationCode);
+
+      final Client client = ClientBuilder.newBuilder().build();
+      Response response = client
+            .target(TARGET_URI)
+            .path(createUrl(
+                  LumeerConst.Security.ORGANIZATION_RESOURCE,
+                  organizationCode,
+                  LumeerConst.Security.ROLES_KEY))
+            .request().buildGet()
+            .invoke();
+
+      List<Role> fromService = response.readEntity(List.class);
+      assertThat(fromFacade.size()).isEqualTo(fromService.size());
```

Comment:
Please, use collection assertions. It is easier to see where the problem is when such assertion fails.
```java
assertThat(fromService).hasSize(fromFacade.size());
```